plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1777,4 +1899,37 @@ private Session broadcastJoinDistribution(boolean dynamicFilteringEnabled)
                 .setSystemProperty(ENABLE_DYNAMIC_FILTERING, Boolean.toString(dynamicFilteringEnabled))
                 .build();
     }
+
+    protected String getTableLocation(String tableName)
+    {
+        return (String) computeScalar("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName);
```

Comment:
Can we use `SHOW CREATE TABLE` result likes Iceberg test? Thus, we don't need to insert rows.

Reply:
Reminder.

Reply:
Addressed!